requirements.txt

```diff
@@ -3,3 +3,7 @@ Flask-Script==2.0.5
 
 # Elasticsearch 1.0
 elasticsearch>=1.0.0,<2.0.0
+
+#Required for testing
```

Comment:
At least one of the other projects (I forget which right now - API?) has a separate requirements-for-test.txt file, which has the extra packages needed to run the tests.
The run_tests script can then install these additional requirements, without them being installed to simply run the app.

It might be nice to follow the same pattern here?